main/src/main.cpp

```diff
@@ -37,6 +38,7 @@ void printUsage(char* argv[]){
     << std::endl << " -s {folder containing bin files with SciFi hit information}"
     << std::endl << " -g {folder containing detector configuration}"
     << std::endl << " -d {folder containing .bin files with MC truth information}"
+    << std::endl << " -a {folder containnig bin files with Muon raw bank informatiom}"
```

Comment:
You are not reading muon raw bank information, but muon common hits, so please change the description.